### DISCUSSION TITLE
🏎️ Announcing MAX 24.1.0 ⚡️

### DISCUSSION THREAD

**Discussion by hogepodge** [modular/modular#1833] [General]
[We're excited to announce](https://www.modular.com/blog/announcing-max-developer-edition-preview) that the Modular Accelerated Xecution (MAX) Platform is [now globally available](https://www.modular.com/max)

Starting first as a preview on Linux systems. MAX is a unified set of tools and libraries that unlock performance, programmability and portability for your AI inference pipelines. Each component of MAX is designed to simplify the process of deploying models to any hardware, delivering the best possible cost-performance ratio for your workloads

Starting with this release, we're updating Mojo 🔥 versioning, with the Mojo version now matching the MAX version. You can find full details on what [the release of MAX means that mean for Mojo🔥 here](https://www.modular.com/blog/max-is-here-what-does-that-mean-for-mojo).

If you're eager to get started with MAX, check out the blog post on [Getting Started with MAX DE](https://www.modular.com/blog/getting-started-with-max-developer-edition).

Thanks to everyone for being a part of the Modular community! We're just getting started, and are excited to share what's next.

**Comment by Brian-M-J** [comment]
>We're working hard to open source the Mojo Standard Library next, using the Apache 2.0 license. [...] We are also planning to open source components of MAX – but will have more to say about that later.

_Nice_

**Comment by hogepodge** [comment]
@Ibrahim758 we love Windows (and Mac, and GPU) users, and all of those platforms are on our roadmap right now. This preview release was intended to give us a start in showing the latest improvements to MAX and Mojo to the world, and we're hard at work on supporting those platforms. I can't share specifics on when they'll be available, but I want you to know that we hear you. 😅
